### ISSUE TITLE
[Automate] Support content packs

### ISSUE THREAD

**Issue by Pathoschild** [Pathoschild/LookupAnything#277]
Add support for Automate content packs which could add custom machines or recipes.

**Comment by Pathoschild** [comment]
Possible `automate.json` with an example furnace recipe:
```js
{
   "Format": "1.0",
   "Machines": [
      {
         "Type": "BigCraftable",
         "ParentSheetIndex": 13, // furnace
         "Recipes": [
            {
               "Input": [
                  { "Type": "Object", "ParentSheetIndex": 378, "Count": 10 }, // copper ore
                  { "Type": "Object", "ParentSheetIndex": 382 } // coal
               ],
               "Output": { "Type": "Object", "ParentSheetIndex": 334 }, // copper bar
               "Minutes": 30
            }
         ]
      }
   ]
}
```

You can also replace the input/output `ParentSheetIndex` with `Category` (though note you can't have dynamic output like fruit => `<fruit name> Wine`).

**Comment by Pathoschild** [comment]
This will be addressed by the Automate API (#119) instead.
